dubbo-common/src/main/java/org/apache/dubbo/common/utils/ClassHelper.java

```diff
@@ -265,26 +264,33 @@ public static boolean isPrimitive(Class<?> type) {
     }
 
     public static Object convertPrimitive(Class<?> type, String value) {
-        if (type == char.class || type == Character.class) {
+        if (value == null) {
```

Comment:
I think 
```
if(value==null || !isNumber(value)) { 
    return null;
}
```

should be good enough for all these, is it it?

Reply:
If you do this, values ​​that are not numeric will also return null. 
for example `value = “true”`

Reply:
ok